packages/jest-resolve-dependencies/src/index.js

```diff
@@ -47,7 +47,9 @@ class DependencyResolver {
         }
         try {
           return this._resolver.resolveModule(file, dependency, options);
-        } catch (e) {}
+        } catch (e) {
+          console.error(e);
```

Comment:
I'm not sure why we fallback to mock module here... I think a better approach would be to just rethrow the error unless we want to fetch the mock.